main.rb

```diff
@@ -1,10 +1,110 @@
-require 'person'
-require 'capitalize'
-require 'trimmer'
-
-person = Person.new(22, 'maximilianus')
-person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+require 'app'
```

Comment:
- [ ] Am unable to run the app `ruby main.rb` due to a load file error, I think the required file you are importing from outside is not in the correct path, the error is pointing to line one of the file and there we go...
<img width="887" alt="1" src="https://github.com/joskalenda/school-library/assets/77843050/b3fe1776-9476-40f7-963b-c1eb5096149d">

- [ ] Check the possible solution, I had to add the  `./` at the beginning of all your require path and it was able to run the app before we proceed with any code check, please make sure you fix this issue

<img width="569" alt="2" src="https://github.com/joskalenda/school-library/assets/77843050/24cd88ca-49ad-4466-a025-a3815162322a">